examples/react-router-example/Gemfile

```diff
@@ -38,5 +38,6 @@ gem 'spring',        group: :development
 # Use debugger
 # gem 'debugger', group: [:development, :test]
 
-gem 'react-rails', '~> 1.0'
-gem 'react-router-rails', '0.13.3.1'
+gem "react-rails", "~> 1.4.0"
+# Using a relative path to use the local gem code in the example app
+gem 'react-router-rails', path: "../../"
```

Comment:
I made this change to use `react-router-rails` from the local working directory two levels up, rather than load it from rubygems. It helps to iterate with the example app using the latest local code rather than a previously released version.